### PR TITLE
Updated equinix tf provider to v2.4.1

### PR DESCRIPTION
**Pull request by devenkulkarni**

### CODE REVIEW

**Comment by devenkulkarni** [comment]
Tested apply/destroy and it worked successfully without any issues.

File: provider.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     equinix = {
       source  = "equinix/equinix"
-      version = "2.3.2"
+      version = "2.4.0"
```
Comment (displague) [inline provider.tf:9]: 2.4.1 fixed a bug in paginating equinix_metal_device datasources, which this module uses

https://github.com/equinix/terraform-provider-equinix/releases/tag/v2.4.1

Comment (dnoland1) [reply provider.tf:9]: @devenkulkarni let's go with v2.4.1

**Review by glovecchi0 — COMMENTED** [review]
It works well.

image

**Review by displague — APPROVED** [review]

**Review by glovecchi0 — APPROVED** [review]

**Pull request merged**